andela/titan-dev-ah-frontend
# #165427095 Design the notification wrapper, view, delete notifications

#### What does this PR do?

- It adds the notification user interface and handles the notification actions

#### Description of Task to be completed?

- Designed the notification UI components
- Handled `get notifications` action
- Added the button to delete the notification
- It is also possible to navigate to the resource that triggered a notification by clicking on it

#### How should this be manually tested?
- After cloning the repository, check out `/ft-create-notification-container-165427095` branch
- Run `yarn start` to start the application then, you shall see the notification button on the top navigation bar with the number the current user's notifications if any.
- You can try to dismiss any notification by hovering on it and click the right `X` button
- If you click inside the notification message, it will take you to the resource page that triggered it and change the notification's state from `unread` to `read` which will be shown by the icon change.

#### Any background context you want to provide?
- Beware that I modified a little bit on tests to use `test()` instead of `it()` for consistency.
- I also added a single space between consecutive tests scenarios to improve their readability.
#### What are the relevant pivotal tracker stories?
- #165427095
